test/integration/topical_event_about_page_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class TopicalEventAboutPageTest < ActionDispatch::IntegrationTest
+  test "topical event about pages" do
+    setup_and_visit_content_item('topical_event_about_page')
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+    assert_has_component_govspeak(@content_item["details"]["body"])
+    assert_has_contents([
+      {text: "Response in the UK", id: "response-in-the-uk"},
+      {text: "Response in Africa", id: "response-in-africa"},
+      {text: "Advice for travellers", id: "advice-for-travellers"}
+    ])
+  end
+
+  test "breadcrumbs show whether a topical event is archived" do
+    @content_item = JSON.parse(get_content_example("topical_event_about_page")).tap do |item|
+      content_store_has_item(item["base_path"], item.to_json)
```

Comment:
Is there much point doing the `tap` call, as opposed to just a sequential statement?

```ruby
@content_item = JSON.parse(get_content_example("topical_event_about_page"))
content_store_has_item(@content_item["base_path"], @content_item.to_json)
```